quinn-udp/Cargo.toml

```diff
@@ -14,14 +14,17 @@ workspace = ".."
 all-features = true
 
 [features]
-default = ["log"]
-# Write logs via the `log` crate when no `tracing` subscriber exists
+default = ["tracing", "log"]
+# Write logs via the `log` crate trough the `tracing` crate when no `tracing`
+# subscriber exists
 log = ["tracing/log"]
+native-log = ["dep:log"]
```

Comment:
Bikeshedding a little, I think I like `direct-log` better? Thoughts?

Reply:
Both fine with me. Latest push updates to `direct-log`.